app/Http/Requests/V1/TimeEntry/TimeEntryAggregateRequest.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Requests\V1\TimeEntry;
+
+use App\Models\Organization;
+use App\Models\Project;
+use App\Models\Tag;
+use App\Models\Task;
+use App\Models\User;
+use Illuminate\Contracts\Validation\ValidationRule;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Support\Facades\Auth;
+use Korridor\LaravelModelValidationRules\Rules\ExistsEloquent;
+
+/**
+ * @property Organization $organization
+ */
+class TimeEntryAggregateRequest extends FormRequest
+{
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, array<string|ValidationRule>>
+     */
+    public function rules(): array
+    {
+        return [
+            'group_1' => [
+                'required_with:group_2',
+                'in:day,week,month,year,user,project,task,client,billable',
```

Comment:
hat es einen grund, dass tags nicht groupable sind? 
bzgl. naming: ich denke grouping und sub_grouping beschreiben besser was hier passiert als group_1 und group_2. weiß nicht ob das die motivation war aber ja sub_sub_grouping wird dann weird sollten wir noch eine dritte ebene machen, aber denk halt eher nicht dass das passieren wird, zumindest hätt ichs bisher in keinem der timetracking tools gesehen...

Reply:
Ein time entry kann mehrere Tags haben, damit wird das groupen by tag etwas verwirrend. Weil entweder du entscheidest, dich pro Eintrag für einen Haupt-Tag der zum Gruppieren verwendet wird oder du gruppierst über alle, aber dann wird der time entry mehrere Male gezählt pro Tag-Gruppe.

Ja, das mit dem `sub_sub_grouping` war der Gedankengang dahinter, aber `sub_group` ist vermutlich nutzerfreundlicher. Werde ich ändern.